src/main/webapp/index.html

```diff
@@ -6,23 +6,41 @@
   <title>Document</title>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
   <script type="module" src="scripts/index-script.js" defer></script>
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="stylesheet" href="styles/homestyle.css">
 </head>
 <body>
   <div id="nav-placeholder"> </div>
 
-  <h1>Welcome</h1>
-  <h2>How to Navigate the site</h2>
-  <iframe src="https://www.youtube.com/embed/dQw4w9WgXcQ" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-  <h2>FOR WHO:</h2>
-  <ul>
-    <li>Seniors looking for tech assistance</li>
-    <li>People who want to connect during the pandemic</li>
-  </ul>
-  <div id="account-functions">
-    <a href="">Log in</a>
-    <p>Don't have an account?</p>
-    <a id="signup" href="">Sign up</a>
+  <div id="header">
+    <h1>Welcome</h1>
+  </div>
+
+  <div id="content">
+    <div id="content-left">
+      <h2>How to Navigate the site:</h2>
```

Comment:
We'll do a thorough content audit before V1, but I'll note these small things if I happen to notice them: Use sentence case here.

Reply:
Changed text to sentence case.